tools/triageHelper/README.md

```diff
@@ -0,0 +1,14 @@
+# A GitHub triage helper for Durable Functions
+
+## What is it?
+
+An serverless app that automatically lists all issues to be triaged across Durable Functions repos.
+
+## How to use it?
+
+This app can be executed on the cloud or locally. Irrespective of the location, you can obtain the list of issues to triage by GET requesting the `<appHost>/api/triage` endpoint.
+
+## Next Steps
+
+The current implementation is subject to hourly rate limits, which means it error out if too many users attempt to run it within a given hour window.
+To make the implementation more reliable, we should register the app with a GitHub API token so that our hourly rate limit can be increased.
```

Comment:
Hmm, I think it will take some time as I haven't done it before :). It's also a little tricky due to the following reasons:

1) I'm not certain that the GitHub token should live under my GitHub account, so I'll need to find a more general user group and register under that. It's not immediately obvious to me what that more general user group will be.

2) Currently the azure app is under my account, I'll need to find a more general user group there too and deploy it there, so we all have management access to it.

I think that setting up these things will have me moving the API token around until we find the final home for this app.

I also presume that the GitHub token stuff would not live in this repo (because that's a secret) so it would have to be published as an app setting on the corresponding azure app. I assume that the only change needed to this code is to conditionally read the token from the app settings (if it exists) and append it to the URL in the right place.

**_For now,_** given that the app may be executed locally and still provide some automation + work around the rate limit that way (if we hit the limit on azure, we can just run it locally), I think it would be worth merging this in at the very least as a local tool.

But no strong feelings on this. If you would strongly prefer that we wait until the app is fully registered on Azure, then we can hold off on this. It will just probably take me some time, maybe a few weeks.

Reply:
I see, that makes sense! Thanks for the explanation! I'm fine with merging this now so I'll approve the PR, but can you create a GitHub issue with the information in your comment so we can track it somewhere? Thanks!

Reply:
Thanks! Created that issue here: https://github.com/Azure/azure-functions-durable-extension/issues/2533